.github/workflows/terraform_drift.yml

```
@@ -0,0 +1,65 @@
+name: Terraform Drift Detection
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 9 * * 1' #Every Monday at 9:00
+    
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  DIR: "src/"
+  
+jobs:
+  terraform-drift-detection:
+    environment: prod 
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```

set an ubuntu version